compiler-rt/lib/asan/scripts/asan_symbolize.py

```diff
@@ -38,6 +37,7 @@
 allow_system_symbolizer = True
 force_system_symbolizer = False
 
+
 # FIXME: merge the code that calls fix_filename().
```

Comment:
is this `FIXME` still accurate? Otherwise, I'd expect this change to have no effect as it is no called as per the comment.

Reply:
Ok, sorry. I don't know anything about the purpose of the function or the `FIXME`.

At the time the `FIXME` was added, the file already contained calls to `fix_filename` from the `symbolize` methods of a few `Symbolizer` subclasses. My guess is that the author hoped these callers could be merged into `super().symbolize()` instead of using a helper function. The implementations have gotten more complex since then, so I doubt the comment is still relevant.